framework/math/quadratures/angular/cylindrical_quadrature.h

```diff
@@ -1,45 +1,44 @@
 #pragma once
 
-#include "framework/math/quadratures/curvilinear_angular_quadrature.h"
-
-#include "framework/math/quadratures/quadrature.h"
+#include "framework/math/quadratures/angular/curvilinear_quadrature.h"
+#include "framework/math/quadratures/spatial/spatial_quadrature.h"
 
 namespace opensn
 {
 
-/** Cylindrical product angular quadrature. */
-class CylindricalAngularQuadrature : public CurvilinearAngularQuadrature
+class CylindricalQuadrature : public CurvilinearQuadrature
 {
-  //  Methods
+private:
+  /** Initialize with one-dimensional quadratures: a polar quadrature and
+   *  a possibly unique azimuthal quadrature for each polar level. */
+  void Initialize(const SpatialQuadrature& quad_polar,
```

Comment:
still confused about the SpatialQuadrature here

Reply:
See comment above.

Reply:
SpatiaQuadrature has been replaced with a new GaussQuadrature base class. Much cleaner, IMHO.